library/std/src/os/unix/fs.rs

```diff
@@ -1100,3 +1100,36 @@ pub fn lchown<P: AsRef<Path>>(dir: P, uid: Option<u32>, gid: Option<u32>) -> io:
 pub fn chroot<P: AsRef<Path>>(dir: P) -> io::Result<()> {
     sys::fs::chroot(dir.as_ref())
 }
+
+/// Create fifo at the specified path with the specified mode.
```

Comment:
Maybe "Create a FIFO special file at the specified path..."

Reply:
Thanks updated